readthedocs/proxito/views/serve.py

```diff
@@ -798,10 +798,11 @@ class ServeError404(SettingsOverrideObject):
     _default_class = ServeError404Base
 
 
-class ServeRobotsTXTBase(ServeDocsMixin, View):
+class ServeRobotsTXTBase(CDNCacheControlMixin, CDNCacheTagsMixin, ServeDocsMixin, View):
 
     # Always cache this view, since it's the same for all users.
     cache_response = True
+    project_cache_tag = "robots.txt"
```

Comment:
It would be good to have a comment explaining why we are defining this here. I assume it's because we need one and only one tag per-project instead of per-version.